internal/server/instance/drivers/edk2/driver_edk2.go

```diff
@@ -207,5 +212,9 @@ func GetArchitectureFirmwarePairsForUsage(hostArch int, usage FirmwareUsage) []F
 
 // GetenvEdk2Path returns the environment variable for overriding the path to use for EDK2 installations.
 func GetenvEdk2Path() string {
-	return os.Getenv("INCUS_EDK2_PATH")
+	if iep := os.Getenv("INCUS_EDK2_PATH"); iep != "" && util.PathExists(iep) {
```

Comment:
Silently ignoring an invalid path seems like a bad idea

Reply:
Would probably be best to turn this into a `(string, error)` and having it fail if the path is specified but doesn't exist.